VideoCallKitQuickStart/ViewController+CallKit.swift

```diff
@@ -191,6 +204,14 @@ extension ViewController {
             // Use the local media that we prepared earlier.
             builder.audioTracks = self.localAudioTrack != nil ? [self.localAudioTrack!] : [TVILocalAudioTrack]()
             builder.videoTracks = self.localVideoTrack != nil ? [self.localVideoTrack!] : [TVILocalVideoTrack]()
+
+            /**
+             * Use the audio device. Please note that the SDK does not support the use of multiple audio devices at the
+             * same time. If you've already connected to a Room, then all future connection attempts must use the same
+             * TVIDefaultAudioDevice as the first Room. Once all the existing Rooms are disconnected you are free to
+             * choose a new audio device for your next connection attempt.
+             */
```

Comment:
@ceaglest do you think mentioning this comment here as well is ok?  We are already mentioning this in ViewController.swift as well

Reply:
I think saving the main comment for the property declaration is fine. I would keep that one, and shorten this to:

`// Use the audio device that we created earlier. All connection attempts will use the same device.`

Or maybe even from the 2nd sentence from that.